README.md

```diff
@@ -2,7 +2,17 @@
 
 ## Purpose
 
-_... help fill this in!_
+* Define and maintain security policies and procedures for:
+  - the core Node.js project
+  - other projects maintained by the Node.js Foundation technical group
+* Define and maintain policies and procedures for the coordination of security concerns within the external Node.js open source ecosystem
```

Comment:
@dougwilson let me rephrase this. There are certain vulnerabilities in the ecosystem that can affect the full Node.js project, and the public image of the Node community. We need a group of people that can help dealing with those, and possibly contain the fallout. It's not imposing, but it is if the ecosystem does *not* fix the issue, then we can help providing either resources or help to do it.

From a end-user point of view, if `split2` has a security or DoS vulnerability, and I am hit by a truck, someone _should_ be notified and deal with the issue.